src/context-selection/section-filter-selector-bar-item/section-filter-selector-bar-item.js

```diff
@@ -18,6 +18,15 @@ export default function SectionFilterSelectorBarItem() {
 
     useOnDependentParamsChange(deselect)
 
+    // select first section if renderAsTabs === true
+    useEffect(() => {
+        const dataSet = dataSetSectionsInfo.data
+        const sections = dataSet?.sections
+        if (dataSet?.renderAsTabs && !sectionFilter && sections?.length) {
+            setSectionFilter(sections[0].id)
+        }
+    }, [dataSetSectionsInfo, sectionFilter])
+
     const shouldComponentRenderNull = useShouldComponentRenderNull()
```

Comment:
I think some of the logic in the `useEffect` is pretty similar to the logic in `useShouldComponentRenderNull`; maybe the latter hook could be moved above the useEffect so that the value could be used there? This seems to work just fine though